From 4a90327b810a68c55b5dd662a594ff47949c2e8d Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Thu, 19 Mar 2020 17:27:33 +0800
Subject: [PATCH] arm64: dts: rockchip: Improve the aclk_cci frequency for
 rk3399

The initialization frequency of cci was adjusted from 300M to 600M.

Change-Id: I36ea20ec84c97f893894687ce4eb7bd021d372a0
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-vop-clk-set.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-vop-clk-set.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-vop-clk-set.dtsi
index f13985cf8b6d..5ed8dac6cf7d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-vop-clk-set.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-vop-clk-set.dtsi
@@ -171,7 +171,7 @@
 			 <400000000>, <200000000>,
 			 <400000000>, <300000000>,
 			 <300000000>, <300000000>,
-			 <300000000>, <300000000>,
+			 <300000000>, <600000000>,/* aclk_cci */
 			 <100000000>, <150000000>,
 			 <150000000>, <400000000>,
 			 <100000000>, <400000000>,
-- 
2.35.3

